MetaMask/metamask-mobile
# Account nickname is always defined

**Description**
The issue is that when the nickname of the account would be empty, it would be non editable.

**Proposed Solution**
The proposed solution it is, when the name it's empty we will assume that the name of the account will be the last name of it.
Example: 
If it was "Account 2" after confirm the empty string on the account name, it will come back to be "Account 2"

Also its proposed, for the users that already have no nickname on their accounts we set the default name "Account" when the user opens the app. 

**Code Impact**
Low

**Test Cases**
Case1:
* Notice the account nickname
* Long press the account nickname
* Edit the name to an empty string
* Will came back to the last account nickname

**Screenshots/Recordings**

https://user-images.githubusercontent.com/46944231/179572818-7b5589f9-b820-48a9-bc10-692f2213229b.mp4


_If applicable, add screenshots and/or recordings to visualize the before and after of your change_

**Issue**

Progresses #4308 

**Checklist**
